server/app/scavenger.py

```diff
@@ -127,6 +127,7 @@ def start_story(message, user, group):
     if not start_clue:
         raise ValueError('Story {} has no clue named "START"'.format(story_code.story_uid))
     group_code = Group.gen_uid()
+    story_code.set_group_uid(group_code)
```

Comment:
Hrmm; what does this mean for the multi-use codes? I suppose they'll just always have the name of the latest group to use them right? I think that's probably fine 👍